redis/3-highly-available/README.md

```diff
@@ -33,3 +33,158 @@ The following variables are supported
 Variable | Description | Expected values | Default
 --- | --- | --- | ----
 REDIS_MASTER_DOWN_AFTER_MILLISECONDS | How many milliseconds to wait before starting a master failover (if two or more redis sentinels agree) | integer (milliseconds) | 5000
+
+### Example Automatic Deployment via continuous-pipe.yml
+
+The following are some example tasks for automating the deployment of a highly available cluster:
+
+```yaml
+variables:
+  REDIS_REPLICAS: 3
+  REDIS_SENTINEL_REPLICAS: 3
+
+tasks:
+  infrastructure:
+    deploy:
+      services:
+        mysql: ~
```

Comment:
Any reason to include MySQL in this example? 🤔

Reply:
Good point, it's to ensure it only happens the first time an environment is created.

Reply:
Is there a better way to do that?

Reply:
I see. Maybe you can use a simple `nginx` container and add a comment to explain why? I read it as "Why the fuck Redis HA needs MySQL?" 😄